tests/access/test_access_constants.py

```diff
@@ -15,19 +15,15 @@ def get_sample(sample_file: str) -> Union[str, bytes]:
 # Access Administration Result Sample Data
 # ============================================================================
 
-# Add Access
-TEST_ADD_ACCESS_RESULT_SUCCESS_XML = get_sample("add_access_result_success.xml")
-TEST_ADD_ACCESS_RESULT_SUCCESS_DICTIONARY = get_sample("add_access_result_success.json")
-TEST_ADD_ACCESS_RESULT_ERROR_XML = get_sample("add_access_result_error.xml")
-TEST_ADD_ACCESS_RESULT_ERROR_DICTIONARY = get_sample("add_access_result_error.json")
-
 # Alter Access
```

Comment:
Change comment:

```python
# Permit Access
```

Reply:
fixed across the board as well as in group connection admin.